ArchiSteamFarm.OfficialPlugins.Monitoring/MonitoringPlugin.cs

```diff
@@ -230,5 +247,53 @@ internal sealed class MonitoringPlugin : OfficialPlugin, IDisposable, IOfficialG
 			},
 			description: "Remaining games to redeem in background per bot"
 		);
+
+		Meter.CreateObservableCounter(
+			$"{MetricNamePrefix}_bot_trades", () => TradeStatistics.SelectMany<KeyValuePair<Bot, TradeStatistics>, Measurement<uint>>(
+				static kv => [
+					new Measurement<uint>(
+						kv.Value.AcceptedOffers,
+						new KeyValuePair<string, object?>(TagNames.BotName, kv.Key.BotName),
+						new KeyValuePair<string, object?>(TagNames.SteamID, kv.Key.SteamID),
+						new KeyValuePair<string, object?>(TagNames.TradeOfferResult, "accepted")
+					),
+					new Measurement<uint>(
+						kv.Value.RejectedOffers,
+						new KeyValuePair<string, object?>(TagNames.BotName, kv.Key.BotName),
+						new KeyValuePair<string, object?>(TagNames.SteamID, kv.Key.SteamID),
+						new KeyValuePair<string, object?>(TagNames.TradeOfferResult, "rejected")
+					),
+					new Measurement<uint>(
+						kv.Value.IgnoredOffers,
+						new KeyValuePair<string, object?>(TagNames.BotName, kv.Key.BotName),
+						new KeyValuePair<string, object?>(TagNames.SteamID, kv.Key.SteamID),
+						new KeyValuePair<string, object?>(TagNames.TradeOfferResult, "ignored")
+					),
+					new Measurement<uint>(
+						kv.Value.BlacklistedOffers,
+						new KeyValuePair<string, object?>(TagNames.BotName, kv.Key.BotName),
+						new KeyValuePair<string, object?>(TagNames.SteamID, kv.Key.SteamID),
+						new KeyValuePair<string, object?>(TagNames.TradeOfferResult, "blacklisted")
+					),
+					new Measurement<uint>(
+						kv.Value.ConfirmedOffers,
+						new KeyValuePair<string, object?>(TagNames.BotName, kv.Key.BotName),
+						new KeyValuePair<string, object?>(TagNames.SteamID, kv.Key.SteamID),
+						new KeyValuePair<string, object?>(TagNames.TradeOfferResult, "2fa_confirmed")
```

Comment:
This is probably misleading, a donation trade offer will be `Confirmed` despite no confirmation being needed, even for people without 2FA.

Consider `automatically_confirmed`, `accepted_and_confirmed` or something similar instead.

Reply:
I'm not sure what you mean exactly. Can you elaborate?

Reply:
> I'm not sure what you mean exactly. Can you elaborate?

`2fa_confirmed` name.